inc/oauth2/namespace.php

```diff
@@ -118,10 +119,25 @@ function attempt_authentication( $user = null ) {
 
 	$is_querying_token = true;
 
-	$local_user = synchronize_user_for_token( $token_value );
-	if ( is_wp_error( $local_user ) ) {
-		$delegated_auth_error = $local_user;
-		return $user;
+	// Locally cache the user for the given access token if it is enabled.
+	if ( defined( 'HM_DELEGATED_AUTH_ACCESS_TOKEN_CACHE_TTL' ) ) {
+		$cached_id = wp_cache_get( $token_value, 'local_user_id_for_token' );
```

Comment:
Do we want to define this as a global group?

Reply:
yeah I think so

Reply:
Added in a new `bootstrap()` function.